src/detectors/DRICH/DRICH.cc

```diff
@@ -24,9 +24,10 @@ extern "C" {
 
     // Digitization
     PhotoMultiplierHitDigiConfig digi_cfg;
-    digi_cfg.seed            = 0;
+    digi_cfg.seed            = 5; // FIXME: set to 0 for a 'unique' seed, but
+                                  // that seems to delay the RNG from actually randomizing
     digi_cfg.hitTimeWindow   = 20.0; // [ns]
-    digi_cfg.timeStep        = 0.0625; // [ns]
+    digi_cfg.timeStep        = 16.0; // [ns^-1]
```

Comment:
I just looked at some TDC spec sheets and I think this quantity is usually expressed as time, not reciprocal. Instead of `step`, maybe use `LSB` or `resolution`, which is how it is described in spec sheets (resolution is also how it is done for silicon tracker digitization).

Reply:
resolved by 5abba9e37f9e56751d69a5c60871e4a67152a4f2